website/blog/2021-12-08-introducing-the-new-relay-compiler.mdx

```diff
@@ -61,15 +63,17 @@ And while we're not there yet, we have made significant achievements on each of
 
 For example, the new compiler ships with support for the new `@required` directive, which will nullify the parent linked field or throw an error if a given subfield is null when read out. This may sound like a trivial quality-of-life improvement, but if half of your component's code is null checks, `@required` starts to look pretty good!
 
-<style type="text/css">{" img { max-height: 600px; }"}</style>
+:::note A component without `@required`
```

Comment:
I thought wrapping these images in admonitions looked a bit better than the current block quote